source/_redirects

```diff
@@ -0,0 +1,10 @@
+/feed                 /feed.xml                                     200
+/tutorial             /guide
+/download             /install
+/try                  https://www.sassmeister.com
+/about                /
+/blog/posts/560719    /blog/dropping-support-for-old-ruby-versions
+/blog/posts/1305238   /blog/dart-sass-is-on-chocolatey
+/blog/posts/1404451   /blog/sass-and-browser-compatibility
+/blog/posts/1909151   /blog/dart-sass-is-in-beta
+/blog/posts/7081811   /blog/ruby-sass-is-deprecated
```

Comment:
I added some of these in just to test them out, but added a new card to make sure we cover all the redirects: https://trello.com/c/xxnQok8e